### PR TITLE
intro: Add asciinema embed

### PR DESCRIPTION
**Pull request by adtac**
Add https://asciinema.org/a/42945?speed=2 to the intro
page.

Fixes https://github.com/coala-analyzer/website/issues/24

### CODE REVIEW

**Comment by sils** [comment]
did you test locally that it works? we don't want our website go down

File: _posts/2000-01-01-intro.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ style: center
 
 # coala - COde AnaLysis Application
 
+<script type="text/javascript" src="https://asciinema.org/a/42945.js" id="asciicast-42945" async></script>
```
Comment (sils) [inline _posts/2000-01-01-intro.md:10]: can you set speed to 2?

**Comment by adtac** [comment]
@sils1297 yeah, it works

**Comment by sils** [comment]
cool :) can we adjust the speed somehow? (Merged this for now since it's probably better than without it)

**Pull request merged**